HW4/main.py

```diff
@@ -0,0 +1,76 @@
+from flask import Flask, render_template
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+
+from db_executor import execute_query
+
+app = Flask(__name__)
+
+
+@app.route("/order_price")
+@use_kwargs(
+    {
+        "country": fields.Str(
+            required=False,
+            load_default=None,
+        )
+    },
+    location="query"
+)
+def order_price(country):
+    query = """
+            SELECT SUM(invoice_items.UnitPrice * invoice_items.Quantity) AS Sales, invoices.BillingCountry AS Country
```

Comment:
better to use group by in both cases but when country is in query use having command